code/modules/mob/dead/observer/observer.dm

```diff
@@ -249,6 +249,7 @@ Works together with spawning an observer, noted above.
 /mob/proc/ghostize(can_reenter_corpse = 1)
 	if(key)
 		if(!cmptext(copytext(key,1,2),"@")) // Skip aghosts.
+			stop_sound_channel(CHANNEL_HEARTBEAT) //Stop heartbeat sounds because You Are A Ghost Now
```

Comment:
It's not meant to be a saying or something, Capitalizing Random Things is something I do for emphasis/to be funny.